Cargo.toml

```diff
@@ -18,6 +18,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 default = ["proc_macro"]
```

Comment:
I wonder if it should be the default hasher or not. @jaemk ?

Reply:
imho, I see little downsides using this as default.